riptable/rt_datetime.py

```diff
@@ -119,9 +119,68 @@
+    DAYS_AT_2000 + (42 * DAYS_PER_YEAR) + (11),
+    DAYS_AT_2000 + (43 * DAYS_PER_YEAR) + (11),
+    DAYS_AT_2000 + (44 * DAYS_PER_YEAR) + (11),
+    DAYS_AT_2000 + (45 * DAYS_PER_YEAR) + (12),
+    DAYS_AT_2000 + (46 * DAYS_PER_YEAR) + (12),
+    DAYS_AT_2000 + (47 * DAYS_PER_YEAR) + (12),
+    DAYS_AT_2000 + (48 * DAYS_PER_YEAR) + (12),
+    DAYS_AT_2000 + (49 * DAYS_PER_YEAR) + (13),
+    DAYS_AT_2000 + (50 * DAYS_PER_YEAR) + (13),  # 2050
+    DAYS_AT_2000 + (51 * DAYS_PER_YEAR) + (13),
+    DAYS_AT_2000 + (52 * DAYS_PER_YEAR) + (13),
+    DAYS_AT_2000 + (53 * DAYS_PER_YEAR) + (14),
+    DAYS_AT_2000 + (54 * DAYS_PER_YEAR) + (14),
+    DAYS_AT_2000 + (55 * DAYS_PER_YEAR) + (14),
+    DAYS_AT_2000 + (56 * DAYS_PER_YEAR) + (14),
+    DAYS_AT_2000 + (57 * DAYS_PER_YEAR) + (15),
+    DAYS_AT_2000 + (58 * DAYS_PER_YEAR) + (15),
+    DAYS_AT_2000 + (59 * DAYS_PER_YEAR) + (15),
+    DAYS_AT_2000 + (60 * DAYS_PER_YEAR) + (15),  # 2060
+    DAYS_AT_2000 + (61 * DAYS_PER_YEAR) + (16),
+    DAYS_AT_2000 + (62 * DAYS_PER_YEAR) + (16),
+    DAYS_AT_2000 + (63 * DAYS_PER_YEAR) + (16),
+    DAYS_AT_2000 + (64 * DAYS_PER_YEAR) + (16),
+    DAYS_AT_2000 + (65 * DAYS_PER_YEAR) + (17),
+    DAYS_AT_2000 + (66 * DAYS_PER_YEAR) + (17),
+    DAYS_AT_2000 + (67 * DAYS_PER_YEAR) + (17),
+    DAYS_AT_2000 + (68 * DAYS_PER_YEAR) + (17),
+    DAYS_AT_2000 + (69 * DAYS_PER_YEAR) + (18),
+    DAYS_AT_2000 + (70 * DAYS_PER_YEAR) + (18),  # 2070
+    DAYS_AT_2000 + (71 * DAYS_PER_YEAR) + (18),
+    DAYS_AT_2000 + (72 * DAYS_PER_YEAR) + (18),
+    DAYS_AT_2000 + (73 * DAYS_PER_YEAR) + (19),
+    DAYS_AT_2000 + (74 * DAYS_PER_YEAR) + (19),
+    DAYS_AT_2000 + (75 * DAYS_PER_YEAR) + (19),
+    DAYS_AT_2000 + (76 * DAYS_PER_YEAR) + (19),
+    DAYS_AT_2000 + (77 * DAYS_PER_YEAR) + (20),
+    DAYS_AT_2000 + (78 * DAYS_PER_YEAR) + (20),
+    DAYS_AT_2000 + (79 * DAYS_PER_YEAR) + (20),
+    DAYS_AT_2000 + (80 * DAYS_PER_YEAR) + (20),  # 2080
+    DAYS_AT_2000 + (81 * DAYS_PER_YEAR) + (21),
+    DAYS_AT_2000 + (82 * DAYS_PER_YEAR) + (21),
+    DAYS_AT_2000 + (83 * DAYS_PER_YEAR) + (21),
+    DAYS_AT_2000 + (84 * DAYS_PER_YEAR) + (21),
+    DAYS_AT_2000 + (85 * DAYS_PER_YEAR) + (22),
+    DAYS_AT_2000 + (86 * DAYS_PER_YEAR) + (22),
+    DAYS_AT_2000 + (87 * DAYS_PER_YEAR) + (22),
+    DAYS_AT_2000 + (88 * DAYS_PER_YEAR) + (22),
+    DAYS_AT_2000 + (89 * DAYS_PER_YEAR) + (23),
+    DAYS_AT_2000 + (90 * DAYS_PER_YEAR) + (23),  # 2090
+    DAYS_AT_2000 + (91 * DAYS_PER_YEAR) + (23),
+    DAYS_AT_2000 + (92 * DAYS_PER_YEAR) + (23),
+    DAYS_AT_2000 + (93 * DAYS_PER_YEAR) + (24),
+    DAYS_AT_2000 + (94 * DAYS_PER_YEAR) + (24),
+    DAYS_AT_2000 + (95 * DAYS_PER_YEAR) + (24),
+    DAYS_AT_2000 + (96 * DAYS_PER_YEAR) + (24),
+    DAYS_AT_2000 + (97 * DAYS_PER_YEAR) + (25),
+    DAYS_AT_2000 + (98 * DAYS_PER_YEAR) + (25),
+    DAYS_AT_2000 + (99 * DAYS_PER_YEAR) + (25),
 ])
 
-# UTC @ midnight, years 1970 - 2040
+# UTC @ midnight, years 1970 - 2099

```

Comment:
Could this array just be created by writing:
```python
UTC_1970_SPLITS = UTC_1970_DAY_SPLITS * NANOS_PER_DAY
```

Reply:
Yes, it can!